### DISCUSSION TITLE
About a solutions manual

### DISCUSSION THREAD

**Discussion by kbrinn** [udlbook/udlbook#209] [Q&A]
Hi Professor Prince-

I graduated from Grad School about two years ago, and recently, I bought a copy of your book Understanding Deep Learning to learn the math Behind Deep Learning that I didn't pick up in school. I may be a little bit rusty on math. I can complete some of the math exercises, but I can't 100% guarantee that my answers are correct. Is there a solutions manual that I could use to help me go through the book? 

Thank you. Have a great week!

**Comment by udlbook** [comment]
Hi Kevin,

The problems with stars have solutions (you can download from
http://udlbook.com).  The remaining solutions are reserved for teachers and
I'm not allowed to send them to you by the terms of my contract.  If you
have a question about a particular problem, then I'm happy to help out
though.  Mail me at ***@***.***

Simon

**Reply by kbrinn** [reply]
I understand. Thank you for the information! 
I didn't get your email address. GitHub placed * characters on it —but if I have any questions, I'll try to message you through one of your social media accounts.

**Comment by udlbook** [comment]
The address is on the front of the PDF at http://udlbook.com

**Reply by kbrinn** [reply]
I found it; thank you!
